### PR TITLE
MGDOBR-964: Re-design deletion components

### PR DESCRIPTION
**Pull request by vpellegrino**
**Task**: https://issues.redhat.com/browse/MGDOBR-964

### CODE REVIEW

**Comment by sjamboro** [comment]
@vpellegrino E2E tests are fixed on the main branch

**Review by manstis — APPROVED** [review]
LGTM :+1:

**Review by sjamboro — APPROVED** [review]
LGTM

File: src/hooks/useBridgesApi/useCanIDeleteBridge.ts [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+        .listProcessors(bridgeId, undefined, 0, 1)
+        .then((response) => {
+          return response.data?.total === 0;
+        })
+        .catch((err) => {
+          if (!axios.isCancel(err)) {
+            setError(err);
+          }
+          return false;
+        });
```
Comment (kelvah) [inline src/hooks/useBridgesApi/useCanIDeleteBridge.ts:32]: Do we need this check here?

**Review by kelvah — APPROVED** [review]
LGTM

File: src/hooks/useBridgesApi/useCanIDeleteBridge.ts (hunk 1 above)
Comment (vpellegrino) [reply src/hooks/useBridgesApi/useCanIDeleteBridge.ts:32]: I think we can get rid of this check. Thank you @kelvah

**Review by sjamboro — APPROVED** [review]

**Pull request merged**